build-status.el

```diff
@@ -375,8 +375,7 @@ Signals an error if the response does not contain an HTTP 200 status code."
 ;;;###autoload
 (define-minor-mode build-status-mode
   "Monitor the build status of the buffer's project."
-  :global t
```

Comment:
BTW, my custom `*build-status` function returns `nil` will caused my custom mode-line error. I don't know how to fix it.